lib/tasks/analyse_javascript.rake

```diff
@@ -0,0 +1,5 @@
+desc 'Analyse Javascript with JSHint and Javascript Code Style checker'
+task :analyse_javascript do
+  puts 'Running Javascript code analysis...'
+  system('npm run js') || raise('Javascript analysis failed')
```

Comment:
In testing this change locally (removing the `raise`) it seemed to continue to run the Jasmine tests afterwards, so leaving this in for now.